File: src/main/java/com/ukm/programkerja/service/InformasiService.java
package com.ukm.programkerja.service;

import com.ukm.programkerja.dto.request.InformasiRequest;
import com.ukm.programkerja.dto.response.InformasiResponse;

import java.util.List;

public interface InformasiService {
    InformasiResponse create(Long userId, InformasiRequest request);
    InformasiResponse update(Long id, InformasiRequest request);
    void delete(Long id);
    List<InformasiResponse> getAll();
}

File: frontend/src/services/informasiService.js
import api from './api'

export default {
  getAll() {
    return api.get('/informasi')
  },
  create(data) {
    return api.post('/informasi', data)
  },
  update(id, data) {
    return api.put(`/informasi/${id}`, data)
  },
  delete(id) {
    return api.delete(`/informasi/${id}`)
  },
}
